wsdl/ver10/schema/metadatastream.xsd

```diff
@@ -389,6 +389,34 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 		</xs:attribute>
 		<xs:anyAttribute processContents="lax"/>
 	</xs:complexType>
+	<xs:complexType name="ObjectSnapshot">
+		<xs:sequence>
+			<xs:element name="ImageRef" type="xs:anyURI" minOccurs="0"/>
+			<xs:element name="Image" type="xs:base64Binary" minOccurs="0"/>
+			<xs:element name="BoundingBox" type="tt:Rectangle" minOccurs="0">
```

Comment:
Why we need this bounding box information, is this image scene image or cropped object image ?